src/Schema/Column/AbstractColumnFactory.php

```diff
@@ -166,13 +168,14 @@ public function fromType(string $type, array $info = []): ColumnInterface
         }
 
         /** @psalm-var class-string<ColumnInterface> $columnClass */
-        $columnClass = $this->mapType($this->columnClassMap, $type, $info)
+        $columnClass = $this->mapType($this->classMap, $type, $info)
             ?? $this->getColumnClass($type, $info);
 
-        $column = new $columnClass($type, ...$info);
+        $columnParams = $info + ($this->classDefaults[$type] ?? []);
```

Comment:
Maybe use `$columnClass` instead of `$type` to set/get default parameters?

`$columnClass` also depends on `$info` and can be different for the same `$type`.

Reply:
> `$columnClass` also depends on `$info` and can be different for the same `$type`.

That's why I used `$type`. For example, one class used for two types, and user want to set different defaults for each type. 

We set column classes by types, so class defaults also must be set by types.